model-optimizer/unit_tests/extensions/ops/dft_signal_size_canonicalization_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+
+import numpy as np
+from generator import generator, generate
+
+from extensions.ops.dft import DFT, IDFT, FFTBase
+from extensions.ops.MatMul import MatMul, transpose
+from mo.front.common.partial_infer.utils import int64_array
+from mo.graph.graph import Node
+from unit_tests.utils.graph import build_graph
```

Comment:
Please, cleanup imports. Many of them are not needed.

Reply:
Fixed.